r/R/utilities_expression.R

```diff
@@ -0,0 +1,62 @@
+#' Returns expression values for selected genes
+#'
+#' @param genes - Must have names and input(ensmbl ids)
+#' @param data
+#'
+#' @return
+#' @export
+#'
+#' @examples
+getExpressionValues <- function(genes, data) {
```

Comment:
instead of having `utilities_x` (like `utilities_development`, `utilities_expression`, etc), why don't you have a folder called "utilities" and inside that folder have "expression.R", "development.R", etc?

Reply:
Also, utilities is a pain to spell, I would use something easier to spell as a name, for example "utils"

Reply:
to be consistent with the python worker, I would actually name the folder not `utilities` or `utils`. I would name it `helpers`.
And in general, I recommend trying to define a structure for those files under `R`, or better follow the structure of the python worker. With lack of structure, it will get very messy when we have even more files (for an example of what happens when not structuring things, look at the utils folder in the UI)

Reply:
Best coding practices in R indicate the usage of a one level folder named R where the code files are set. We based this structure in some major R packages. For example:[ dplyr](https://github.com/tidyverse/dplyr/tree/main/R).

Dplyr itself uses "utils", I based this on another package that used "utilities". I think this is a good idea, I will rename

Reply:
Interesting, I didn't know that. Do you know the reason why?